src/nvim/api/ui_events.in.h

```diff
@@ -123,6 +123,10 @@ void win_viewport(Integer grid, Window win, Integer topline,
                   Integer line_count)
   FUNC_API_SINCE(7) FUNC_API_REMOTE_ONLY;
 
+void win_extmarks(Window win, Integer ns_id, Integer mark_id, Integer row,
```

Comment:
For consistency, this should take `Integer grid` before `win`

Reply:
Done. This accidentally fixed some multi-window same buf sync problems I mentioned earlier.